hic_qc.py

```diff
@@ -25,36 +25,29 @@
 import pdfkit
 import markdown as md
 from scipy import optimize
```

Comment:
I think the scipy optimize import can be removed if we're no longer fitting the saturation curve. The environment would be quite a bit simpler without it.